addons/dark-www/addon.json

```diff
@@ -2305,7 +2324,7 @@
         "blue": "#252c37",
         "input": "#202020",
         "button": "#4d97ff",
-        "link": "#4d97ff",
+        "link": "#67a7ff",
```

Comment:
I don't think this change is necessary - users who decide to revert to the blue color probably don't care about contrast. But if we do change this color, the same change should be made to the blue version of 3.Darker.